src/CodepageTranslator.cc

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2020 Peter Kessen
+ *
+ * This file is part of dotprint.
+ *
+ * dotprint is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * dotprint is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with dotprint. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "CodepageTranslator.h"
+#include <fstream>
+#include <iostream>
+#include <iomanip>
+
+CodepageTranslator::CodepageTranslator()
+{
+}
+
+CodepageTranslator::~CodepageTranslator()
+{
+}
+
+void CodepageTranslator::loadTable(std::string const& tableName)
```

Comment:
The code is written so that we can also add a different implementation of the `ICodepageTranslator` which uses the libiconv. So i think this was already covered by this.

Reply:
Yes, with the interface, it is possible.